shop_app.rb

```diff
@@ -0,0 +1,47 @@
+require "bundler/setup"
+require "sinatra/base"
+require "sinatra/reloader"
+
+Dir["./lib/**/*.rb"].each { |ruby_file| require ruby_file }
+require "./db/seeds.rb"
+
+module Shop
+  class App < Sinatra::Base
```

Comment:
The convention is to put `app.rb` into `shop` directory if you have `module Shop` namespace.